Resources/doc/internals/configuring_the_http_client.md

```diff
@@ -27,4 +27,93 @@ hwi_oauth:
 
 If you don't want to use the HTTPlug Bundle, the client and factory services **must be specified**.
 
+## Difference since 0.6+ version
+
+On v0.5 and below, you was able to configure your http client like that:
```

Comment:
you were

Reply:
Not my documentation, but I tend to pay attention to details: I usually like being consistent about the same expressions: http client vs HTTP Client, etc.